docs/spec.md

```diff
@@ -1135,85 +1136,253 @@ in Evy. The call arguments must be listed individually.
 
 ## Break and Return
 
-`break` and `return` are terminating statements. They interrupt the
-regular flow of control. `break` is used to exit from the inner-most
-loop body. `return` is used to exit from a function and may be followed
-by an expression whose value is returned by the function call.
+`break` and `return` are _terminating statements_ in Evy. They interrupt
+the regular flow of control.
+
+- `break` is used to exit from the innermost loop body. This means that
+  it will skip the rest of the loop body and continue with the next
+  statement after the loop.
+- `return` is used to exit from a function. It can be followed by an
+  expression whose value is returned by the function call.
+
+For example, the following code shows how the `break` statement can be
+used to exit from a loop:
+
+```evy
+for x := range 2
+    y := 0
+    while y < 10
+        if y == 2
+            print "break" y
+            break
+        end
+        print "no break" y
+        y = y + 1
+    end
+    print "x" x "y" y
+    print
+end
+```
+
+This code will print the following output:
+
+```evy:output
+no break 0
+no break 1
+break 2
+x 0 y 2
+
+no break 0
+no break 1
+break 2
+x 1 y 2
+
+```
+
+As you can see, the `break` statement causes the loop to exit when the
+value of `y` is equal to 2. The next statement after the loop is then
+executed.
```

Comment:
Since you used nested loops in the example, perhaps it deserves a mention in the text?

Reply:
yeah, good point. I've mentioned it in the text now.